autoload/fuzzyy/utils/selector.vim

```diff
@@ -114,6 +114,31 @@ function IsBinaryBlob(path)
     return v:false
 endfunction
 
+# Get filetype from modelines, use when not detected via filetypedetect autocmd
+export def FTDetectModelines(content: list<string>): string
+    if empty(content)
+        return ''
+    endif
+    try
+        var modelines = len(content) >= &modelines ? &modelines : len(content)
+        var pattern = '\M\C\s\?\(V\|v\)im\?:\.\*\(ft\|filetype\)=\w\+'
```

Comment:
Note to self, fix pattern, spec in docs only allows `Vim`, `vim`, and `vi` (ignoring ex)